build/controller/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 The Kubernetes Authors.
+# Copyright 2023 The Kubernetes Authors.
```

Comment:
In k8s' codebase, it's conventional to keep the original date instead of updating it for each year.